src/NBS.Appointments.Service.Api.Tests/RescheduleAppointmentApiTests.cs

```diff
@@ -0,0 +1,52 @@
+﻿using FluentAssertions;
+using NBS.Appointments.Service.Models;
+using System.Net;
+using System.Text;
+using System.Text.Json;
+using Xunit;
+
+namespace NBS.Appointments.Service.Api.Tests
+{
+    public class RescheduleAppointmentApiTests
+    {
+        private readonly HttpClient _httpClient = new();
+        private const string Endpoint = "http://localhost:4000/appointment/reschedule";
```

Comment:
I wonder if we want to have base class or something that pulls the main host from config? And then just specify the Endpoint specific path on a derived property or something like this? Just so if we change the port or whatever we don't have to update everywhere?